PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,5 +1,11 @@
 ## What does this change?
+<!-- A PR should have enough detail to be understandable far in the future. e.g what is the problem/why is the change needed, how does it solve it and any questions or points of discussion. Prefer copying information from a Trello card over linking to it; the card may not always exist and reviewers may not have access to the board. -->
 
 ## How can we measure success?
+<!-- Do you expect errors to decrease? Do you expect user journeys to be simplified? What can be used to prove this? A filtered view of logs or analytics, etc? -->
+
+## What is the cost of failure?
+<!-- Is this a critical path? Does failure costs money, violates law, ruins reputation? Do we need to alarm on failure? -->
```

Comment:
I am not sure if I am keen on this session being added to the default template. I would rather have a section that encourages a focus on risk and safeguards:

------------
## What are the potential risks and how can they be mitigated?
<!-- Is this a critical path? Do we need to alarm on failure? -->
------------

A couple of points just regarding the grammar of this section: 
`costs` -> `cost` 
`violates law` -> `violate laws`

Reply:
Also `ruin reputation` for the same reasons
```suggestion
<!-- Is this a critical path? Does failure cost money, violate laws, ruin reputation? Do we need to alarm on failure? -->
```